.changeset/lucky-eels-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'nextra-theme-docs': patch
+---
+
+dd
```

Comment:
```suggestion
fix uncrawable links in sidebar, use button when link don't have href because it impacts on SEO

remove useless prop children frm <FileTree.File /> component
```